db/schema.rb

```diff
@@ -129,7 +129,9 @@
     t.boolean "subject_to_formal_performance_action"
     t.integer "eligible_itt_subject"
     t.boolean "teaching_subject_now"
-    t.integer "itt_academic_year"
+    t.string "itt_academic_year", limit: 9
+    t.boolean "postgraduate_masters_loan"
```

Comment:
Weird, where did loans comes from?

Reply:
@nomasprime This came from the ticket to Add Postgraduate Loans to both M+P and TSLR journeys. Originally the attributes were on the ECP eligibility model, having followed how `loan_amount` works where its on an eligibility model. It always felt wrong, and needed moving. ECP-1130 as an enabled for ECP-446.
Does it not show up in `git blame`?

Reply:
Removed.